com.unity.multiplayer.mlapi/Runtime/Messaging/RpcQueue/RpcQueueProcessor.cs

```diff
@@ -28,21 +27,23 @@ internal class RpcQueueProcessor
         //Temporary place to keep internal MLAPI messages
         private readonly List<RpcFrameQueueItem> m_InternalMLAPISendQueue = new List<RpcFrameQueueItem>();
 
+        //The RpcQueueContainer that is associated with this RpcQueueProcessor
+        internal RpcQueueContainer m_RpcQueueContainer;
+
         /// <summary>
         /// ProcessReceiveQueue
         /// Public facing interface method to start processing all RPCs in the current inbound frame
         /// </summary>
-        public void ProcessReceiveQueue(NetworkUpdateStage currentStage)
+        public void ProcessReceiveQueue(NetworkUpdateStage currentStage, bool isTesting)
```

Comment:
I'm not super crazy about having these parameters in the public API unless we can explain when users should use them.  At the very least, can isTesting have a default?  Does this have to be public?

Reply:
Hmm, that is a good point. 
The class is internal... but I guess one could expose that... and try to create their own version...

Reply:
Oh, apologies, I didn't mean to suggest we *want* it exposed.  If by virtue of this class being internal then non-MLAPI SDK code can't call it then I'm satisfied.